integration-tests/steps/helpers.py

```diff
@@ -73,7 +73,7 @@ def get_cloudformation_stack_name(context, stack_name):
 
 
 def retry_boto_call(func, *args, **kwargs):
-    delay = 2
+    delay = 5
```

Comment:
This change should help resolve some of the intermittent throttling errors we get in our integration tests.